server/stream.go

```diff
@@ -181,13 +195,14 @@ func (a *Account) AddStreamWithStore(config *StreamConfig, fsConfig *FileStoreCo
 	// Setup our internal send go routine.
 	mset.setupSendCapabilities()
 
-	// Create our pubAck here. This will be reused and for +OK will contain JSON
-	// for stream name and sequence.
-	longestSeq := strconv.FormatUint(math.MaxUint64, 10)
-	lpubAck := len(OK) + len(cfg.Name) + len("{\"stream\": ,\"seq\": }") + len(longestSeq)
-	mset.pubAck = make([]byte, 0, lpubAck)
-	mset.pubAck = append(mset.pubAck, OK...)
-	mset.pubAck = append(mset.pubAck, fmt.Sprintf(" {\"stream\": %q, \"seq\": ", cfg.Name)...)
+	// Create our pubAck template here. Better than json marshal each time on success.
+	b, _ := json.Marshal(&JSPubAckResponse{
```

Comment:
So as it stands, this is breaking "clients" that check for +OK in puback response. I don't think that there were client releases that have that yet, but just checking with you guys. (The GO client's jetstream.go does check for +OK, but that is in main branch but not released yet, right? @ripienaar)
May also have to check with Colin, Alberto, etc.. if my understanding is correct.

Reply:
lots in jsm, but thats fine.

Reply:
Just discovered it in java, all good, will fix.  Building the main branch for CI detects breaking changes quickly.  :)